src/index.html

```diff
@@ -6,7 +6,169 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar
+    calendar--start-day
+    calendar--month-length
+    ">
```

Comment:
also, you miss modifiers like these
```suggestion
    <div class="calendar
    calendar--start-day-fri
    calendar--month-length-28
    ">
```